data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/ScalarDbMode.java

```diff
@@ -0,0 +1,16 @@
+package com.scalar.db.dataloader.cli;
+
+/**
+ * @deprecated As of release 3.17.0 This enum is deprecated and will be removed in release 4.0.0.
+ *     The behavior is now determined by the transaction manager configuration in the ScalarDB
+ *     properties file. This enum is kept only for backward compatibility with the deprecated --mode
+ *     CLI option.
+ */
+@Deprecated
+public enum ScalarDbMode {
```

Comment:
While it may seem strange to create a new class that is deprecated right away, this class is added to the cli code as the original class, and its enums have been renamed in the data loader core code for clarity.